crates/indexer/src/models/coin_models/coin_activities.rs

```diff
@@ -185,6 +187,7 @@ impl CoinActivity {
                     txn_info.block_height.unwrap().0 as i64,
                     &entry_function_id_str,
                     txn_timestamp,
+                    index as i64,
```

Comment:
Since this is u64, should we explicitly handle the case where it's greater than i64::max?

Reply:
index will never be greater than i64. Think about it, the index is the position in the array. If we have an array with a billion+ length we have a bigger problem lol.